eng/common/Version-Parser.psm1

```diff
@@ -0,0 +1,8 @@
+﻿$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?))"
```

Comment:
I honestly would keep this function in the Update-Change-Log.ps1 file until we actually have another user of it.